README.md

```diff
@@ -53,31 +70,46 @@ For example, this will check `my-crate-api` and `my-crate-core`:
   with:
     package: my-crate-api, my-crate-core
 ```
-And this will process all crates from the current workspace except `my-crate-tests`:
+
+Inputs `package` and `manifest-path` might be used together:
+```yaml
+- name: Check semver for my-crate from my-workspace
+  uses: obi1kenobi/cargo-semver-checks-action@v2
+  with:
+    manifest-path: semver/my-workspace/Cargo.toml  # or just semver/my-workspace/
+    package: my-crate
+```
+
+### Exclude crates from being checked
+This will process all crates from the current workspace except `my-crate-tests`:
 ```yaml
 - name: Check semver for all crates except my-crate-tests
   uses: obi1kenobi/cargo-semver-checks-action@v2
   with:
     exclude: my-crate-tests
 ```
 
-If the action is not run inside the workspace root directory, you again have to specify the path to its `Cargo.toml` file:
+### Use toolchain other than `stable`
+By default, the actions installs (if necessary) and then uses the `stable` toolchain regardless of local overrides and the `rust-toolchain.toml` file. You can force using a different toolchain using `rust-toolchain`:
```

Comment:
The fact that we use `stable` by default even ignoring `rust-toolchain.toml` is also probably worth mentioning in the `Basic usage` section, with a link down to here in case someone wants to change it.